### PR TITLE
Add bypass cache to get chat config

### PR DESCRIPTION
**Pull request by Tsarar**
image

### CODE REVIEW

File: src/Interfaces/ISDK.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import IReconnectableChatsParams from "../Interfaces/IReconnectableChatsParams";
 import FetchChatTokenResponse from "../Model/FetchChatTokenResponse";
 
 export default interface ISDK {
-  getChatConfig(requestId: string): Promise<object>;
+  getChatConfig(requestId: string, bypassCache: boolean): Promise<object>;
```
Comment (grantborthwick) [inline src/Interfaces/ISDK.ts:6]: Should we default this to false so consumers of this don't have to update their code to explicitly specify false here?

Comment (Tsarar) [reply src/Interfaces/ISDK.ts:6]: I believe that we are doing this in the SDK.ts and this is ok in the type definition
Should I add this to .d.ts?

Comment (nishan) [reply src/Interfaces/ISDK.ts:6]: Yes, we don't want to change the interface as much as possible.

Comment (Tsarar) [reply src/Interfaces/ISDK.ts:6]: Fixed

**Pull request merged**